android-json-form-wizard/src/test/java/com/vijay/jsonwizard/utils/NativeFormLangUtilsTest.java

```diff
@@ -118,4 +119,12 @@ public void testResourceBundleWithPropertiesFromDbIsEmptyWhenClientFormDoesntExi
         ResourceBundle mlsResourceBundle = NativeFormLangUtils.getResourceBundleFromRepository(RuntimeEnvironment.application, interpolatedJsonForm);
         assertFalse(mlsResourceBundle.getKeys().hasMoreElements());
     }
+
+    @Test
+    public void testPerformFormsTranslationShouldReturnCorrectValue() {
+        NativeFormLibrary.getInstance().setPerformFormsTranslation(true);
+        assertTrue(NativeFormLibrary.getInstance().isPerformFormsTranslation());
+        NativeFormLibrary.getInstance().setPerformFormsTranslation(false);
+        assertFalse(NativeFormLibrary.getInstance().isPerformFormsTranslation());
```

Comment:
Could you add assertions to verify that a sample form is actually translated when the values are set to `true` and that it is left untranslated when the value is false?

Reply:
Test cases added

Reply:
Refer JsonFormActivityTest.java

Reply:
Also test that the intent value can override the global value per form i.e. 

1. set global to true and set intent to false -> should not translate form
2. set global to false and set intent to true -> should translate form

Reply:
Required test cases added.